src/deviation.jl

```diff
@@ -51,7 +82,14 @@ function L1dist{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T})
     return r
 end
 
+
 # Linf distance
+"""
+    Linfdist(a, b)
+
+Compute the L∞ distance, also called the Chebyshev distance, between
+two arrays: `max|a-b|`.
```

Comment:
This is the correct math notation as far as I know (at least it's how I learned it) but I'm wondering if maybe describing these functions in terms of equivalent Julia code would be better, e.g. `maximum(abs(a - b))`. What do you think?